tools/install_travis.sh

```diff
@@ -57,7 +57,7 @@ else
     git clone -q https://github.com/lscsoft/lalsuite.git
     cd lalsuite
     # see PR #1067 for the choice of commit
```

Comment:
Can somebody update or remove this comment, since the git hash has been changed?